bitrise.yml

```diff
@@ -3,7 +3,7 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
```

Comment:
pls update format_version

Reply:
pls change test workflows to the new test steps

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/377e74466fa3d4cf0be83992a69b668b7a7b878f